pkgs/by-name/po/pollymc-unwrapped/package.nix

```diff
@@ -0,0 +1,118 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  cmark,
+  darwin,
+  extra-cmake-modules,
+  gamemode,
+  ghc_filesystem,
+  jdk17,
+  kdePackages,
+  ninja,
+  nix-update-script,
+  stripJavaArchivesHook,
+  tomlplusplus,
+  zlib,
+
+  msaClientID ? null,
+  gamemodeSupport ? stdenv.isLinux,
+}:
+
+let
+  libnbtplusplus = fetchFromGitHub {
+    owner = "PrismLauncher";
+    repo = "libnbtplusplus";
+    rev = "a5e8fd52b8bf4ab5d5bcc042b2a247867589985f";
+    hash = "sha256-A5kTgICnx+Qdq3Fir/bKTfdTt/T1NQP2SC+nhN1ENug=";
+
+    # need to build binary
+    # and this is the best fork
+    # from libnbt++ forks
+  };
+in
+
+assert lib.assertMsg (
+  gamemodeSupport -> stdenv.isLinux
+) "gamemodeSupport is only available on Linux.";
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "pollymc-unwrapped";
+  version = "8.0";
+
+  src = fetchFromGitHub {
+    owner = "fn2006";
+    repo = "PollyMC";
+    url = "https://github.com/fn2006/PollyMC/releases/download/8.0/PollyMC-8.0.tar.gz";
```

Comment:
```suggestion
```

Reply:
will this work? on my local machine not including url with repo caused to "404" and wrong url picking.

Reply:
It *should* work. The github fetcher uses the `owner` and `repo` to construct itself a url to fetch from. It shouldn't be necessary to set it. The `rev` looks correct too.

Reply:
'k, i'll commit and do force push as soon as i get back to home.